utils/management/commands/add_admin_user.py

```diff
@@ -0,0 +1,24 @@
+from django.core.management.base import BaseCommand
+
+from users.models import User
```

Comment:
I think, it is more advisable to use `get_user_model` for referencing the user model:
```python
from django.contrib.auth import get_user_model

...

if not get_user_model().objects.filter(username=kwargs["username"]).count():
    get_user_model().objects.create_superuser(
        kwargs["username"], kwargs["email"], kwargs["password"]
    )
```

Reply:
Well, true that, it is better, I'll fix.